tests/Testcontainers.Db2.Tests/Testcontainers.Db2.Tests.targets

```diff
@@ -0,0 +1,22 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+    <PropertyGroup>
+        <RunSettingsFilePath>$(MSBuildProjectDirectory)/linux.runsettings</RunSettingsFilePath>
+    </PropertyGroup>
+    <Target Name="SetEnvironmentVariables" BeforeTargets="RunTests;VSTest">
+        <XmlPoke
+            XmlInputPath="linux.runsettings"
+            Value="$(MSBuildProjectDirectory)/$(OutputPath)clidriver/lib:$(MSBuildProjectDirectory)/$(OutputPath)clidriver/lib/icc"
+            Query="/RunSettings/RunConfiguration/EnvironmentVariables/LD_LIBRARY_PATH"
+        />
+        <XmlPoke
+            XmlInputPath="linux.runsettings"
+            Value="$(MSBuildProjectDirectory)/$(OutputPath)clidriver/bin"
+            Query="/RunSettings/RunConfiguration/EnvironmentVariables/PATH"
+        />
+        <XmlPoke
+            XmlInputPath="linux.runsettings"
+            Value="$(MSBuildProjectDirectory)/$(OutputPath)clidriver"
+            Query="/RunSettings/RunConfiguration/EnvironmentVariables/DB2_CLI_DRIVER_INSTALL_PATH"
+        />
```

Comment:
I moved these parts to a separate targets file. Can't we set the values directly in the `runsettings` file? (I'm not sure if the actual base path is `$(MSBuildProjectDirectory)/$(OutputPath)` already; and we can simply use the relative path).

Reply:
I’m also not sure but to me it looked like the environment variables need to have an absolute path that’s why I implemented it in this way in order to use variables from MSBuild for the absolute path construction.